docs/guide/dev/deep_dive/errors/README.md

```diff
@@ -23,16 +23,22 @@ This article contains the list of CKEditor error codes and their explanation. Th
 * Description: Incorrect {@linkapi CKEDITOR.config#autoEmbed_widget } value. No widget definition found.
 * Additional data: None.
 
-## cloudservices-no-url
+## cloudservices-no-upload-url
 
 * Location: `plugins/cloudservices/plugin.js`
-* Description: Upload URL for the [Cloud Services](https://ckeditor.com/cke4/addon/cloudservices) plugin was not defined.
+* Description: {@linkapi CKEDITOR.config.cloudServices_uploadUrl} for the [CKEditor Cloud Services](https://ckeditor.com/cke4/addon/cloudservices) plugin was not specified.
+* Additional data: None.
+
+## cloudservices-no-token-url
+
+* Location: `plugins/cloudservices/plugin.js`
+* Description: The {@linkapi CKEDITOR.config.cloudServices_tokenUrl} config was not specified.
```

Comment:
This message is not coherent with the previous one, yet both of them are about the same thing: missing config variable.